### DISCUSSION TITLE
does mitsuba allow atomic operations in path Loop

### DISCUSSION THREAD

**Discussion by GensokyoLover** [mitsuba-renderer/mitsuba3#641] [Q&A]
when i use dr::scatter_reduce,which is called by ImageBlock::put in path Loop, the program can not be compiled by drjit. So does mitsuba support atomic operations in Loop? Thank you for your time

**Comment by njroussel** [comment]
I'll answer over here: https://github.com/mitsuba-renderer/mitsuba3/issues/633
